spec/models/comment_spec.rb

```diff
@@ -151,6 +151,28 @@
     }.to change { author.reload.karma }.by(-4)
   end
 
+  describe "performance" do
+    let(:story) { create(:story) }
+    let(:author) { create(:user) }
+
+    it "uses precomputed score on initial insertion" do
+      comment = Comment.build(user: author, story: story, comment: "cool story!")
+      comment.assign_initial_confidence
+
+      expect {
+        expect {
+          expect {
+            expect {
+              comment.update_score_and_recalculate! 0, 0
```

Comment:
The idea here is to trigger the DB side computation, and ensure the result is the same than what we precomputed.